enclave/src/openfinex/openfinex_client.rs

```diff
@@ -74,6 +75,7 @@ impl TcpClient {
             sendable_plaintext: Vec::new(),
             response_handler,
             markets_request_sender,
+            payload_string_buffer: String::new(),
```

Comment:
When does this get cleared/reset? Is there no need to do that?

Reply:
Yes, there is a need to do that, because the frame cotinuation could go on for several frames - until a fin bit is supplied. So until the fin bit, we need to append texts without resetting it.
We then could clear the string after the fin bit is supplied or when the next TextOp is without a fin bit is supplied (i.e. when we need to cache text again).
I've currently chosen the way cleansing the cache after the last fin bit:
https://github.com/Polkadex-Substrate/polkadexTEE-worker/blob/add-continuation_op/enclave/src/openfinex/openfinex_client.rs#L194

Reply:
ah okay, got it - didn't see that clear in the code before 👍